common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/azureactivedirectory/AzureActiveDirectory.java

```diff
@@ -233,6 +247,17 @@ public static synchronized void performCloudDiscovery()
         }
     }
 
+    /**
+     * Ensures that cloud discovery has been completed. If not, it will perform cloud discovery.
+     */
+    public static synchronized void ensureCloudDiscoveryComplete() throws ClientException {
+        final String methodTag = TAG + ":ensureCloudDiscoveryComplete";
+        if (!sIsInitialized) {
```

Comment:
why is this variable prefixed with a s?

Reply:
because it is static. The variable already exists and for same purpose I am using here.